docs/source/models/intervention_models/neonatal/cpap_intervention.rst

```diff
@@ -110,11 +108,31 @@ The following decision tree shows all of the paths from delivery facility choice
 
 where :math:`p(\text{RDS})` is the probability of dying from Preterm with RDS in the general population, and :math:`p(\text{RDS}|\text{CPAP})` and :math:`p(\text{RDS}|\text{no CPAP})` are the probability of dying from Preterm with RDS in setting with and without access to CPAP.  For each path through the decision tree, :math:`p(\text{path})` is the probability of that path; for example the path that includes the edges labeled BEmONC and unavailable occurs with probability that the birth is in a BEmONC facility times the probability that the facility has CPAP available (7.5% in Ethiopia in 2016)
 
-When we fill in the location-specific values for delivery facility rates, CPAP coverage, relative risk of mortality with CPAP access, and mortality probability (which is also age-specific), this becomes a system of two linear equations have two unknown (p_RDS_w and p_RDS_wo), which we can solve analytically.
+When we fill in the location-specific values for delivery facility rates, CPAP coverage by facility type, relative risk of mortality with CPAP access, and mortality probability (which is also age-specific), this becomes a system of two linear equations with two unknowns (p_RDS_w and p_RDS_wo), which we can solve analytically.
```

Comment:
It is still not obvious to me how we are supposed to get the value of $p(\text{RDS})$, as we don't explain it anywhere (see [this comment](https://github.com/ihmeuw/vivarium_research/pull/1590#discussion_r1945539294)).

Reply:
It still seems to me like computing $p(\text{RDS})$ and going through all the trouble of solving these equations is overkill for computing the PAF, when all we need is $p(\text{no CPAP})$.